data/fixtures/scopes/r/name.assignment.scope

```diff
@@ -0,0 +1,34 @@
+hello <- "world"
+hello = "world"
+---
+
+[#1 Content] =
+[#1 Domain] = 0:0-0:5
+  >-----<
+0| hello <- "world"
+
+[#1 Removal] = 0:0-0:6
+  >------<
+0| hello <- "world"
+
+[#1 Trailing delimiter] = 0:5-0:6
+       >-<
+0| hello <- "world"
```

Comment:
` <- ` should be part off the trailing range. Have a look at how we do this in other languages.